### ISSUE TITLE
Feature Request: send fewer notifications

### ISSUE THREAD

**Issue by moqmar** [crazy-max/diun#10]
I can think of two ways here to make the notifications more overseeable:
- Sending all notifications during one cron check at once (instead of one email for multiple tags of the same repository, like it is right now)
- Sending only one email per day/week

I'd prefer this to be configurable, maybe by adding a `summarize` option to each notifier, which can then be set to `false` (don't change anything), `true` (summarize per cron check) or a cron specification like `0 0 18 * * *` (send a summary once per day at 6pm).

**Comment by RezzZ** [comment]
Why not send just _one_ notification per new docker image, unless there is again a newer image than the last one reported? Only need to keep track of the last notification send for each running container, to prevent duplicate notifications.

**Comment by Jonn3y** [comment]
Would love to see something like that too!

One E-Mail with all Containers that could be updated would be great. Like:
"There are some updates for your running docker containers available:"
| Installed  | Avaiable |
| ------------- | ------------- |
| Nextcloud 7 - id abcdefghijklmnop...  | Nextcloud 8 - id ...  |
| mySQL 5.5  | mySQL 5.7  |
| ...  | ...  |
| some old container  | some newer container  |

And even when i got a notification listing a container like nextcloud a day/week before, it would be awesome to list it again until I update it. So diun would remind me with an awesome TODO list per time interval.

Greetings!

**Comment by Jonn3y** [comment]
Any updates on this?

**Comment by crazy-max** [comment]
@Jonn3y A lot of other projects in progress but I will soon unravel all these issues and prepare the arrival of v3 ;)

**Comment by kirsch33** [comment]
i too would be interested in having this feature added. way too many emails but i love the image.

**Comment by muxa** [comment]
They way I'd like to reduce notifications is to not report irrelevant tags. Currently in reports all tags. 

What I'd like to see if only report changes to the tag that my container is based on, e.g. `latest`. I don't want to see `edge` or `dev` etc. 

To make notification more meaningful is to list all tags that match the SHA. For example my imaginary container running `latest` tag, which also has a version tag `1.0.0`. Then when scanning for changes if `latest` changes, find all the tags matching the SHA and list them together, e.g. `1.0.1`. This way I can see that there's an update available (v1.0.1)

**Comment by lepicodon** [comment]
Hi,
I was wondering if there was anything new reg. grouping all email notifications or sending a "simple summary" instead of one email per tag?
I haven't found anything in the docs on that matter, even with the new releases since this issue was created.
Thanks!

**Comment by JSylvia007** [comment]
I'm also curious if there's any movement on this.  Every Saturday I get about 2-dozen emails for all the updates that have happened.  It would be really nice if this were just one email.

**Comment by talz13** [comment]
Yes, I currently subscribe via Pushover, and the smattering of messages sometimes gets lost between my devices. It would be nice to have a single summary message or email, containing all of the updates, rather than individual messages for each.

**Comment by slaszt** [comment]
Would really love an update on this, please. This feature and #1168 would be extremely useful for me and, presumably, many other Diun users.
